CHANGELOG.md

```diff
@@ -30,6 +30,327 @@ More information and tips:
 docs/releases/Hotfix-Documentation-Best-Practices.md
 -->
 
+## Flutter 3.38 Changes
+
+### [3.38.0-0.2.pre](https://github.com/flutter/flutter/releases/tag/3.38.0-0.2.pre)
+
+- [flutter/176462](https://github.com/flutter/flutter/issues/176462) Improves error message when building for iOS fails due to precompiled headers cache error.
+- [flutter/176206](https://github.com/flutter/flutter/issues/176206) Adds MacOS warning that wireless debugging may be slow on iOS 26 that is forwarded to DAP clients, e.g. VSCode
+- [flutter/177037](https://github.com/flutter/flutter/issues/177037) Allows some Android apps that use dynamic modules to build from Android Studio.
+- [flutter/177320](https://github.com/flutter/flutter/issues/177320) Support most recently published Android AGP/KGP.Java/Gradle dependencies.
+- [flutter/176310](https://github.com/flutter/flutter/issues/176310) For Flutter web applications run with `--machine`, ensure `app.dtd` event is emitted.
+- [flutter/177401](https://github.com/flutter/flutter/pull/177401) Fixes a macOS text input crash caused by down-casting the string argument from Any to a NSString.
+- [flutter/177308](https://github.com/flutter/flutter/pull/177308) Configures `FfiNative` resolver on dart:io.
+- [flutter/176360](https://github.com/flutter/flutter/issues/176360) Fixes accessibility events regression on Linux which makes apps not announced with screen reader.
+-[flutter/174791](https://github.com/flutter/flutter/issues/174791) Fixes Flutter web hot reload/restart crashes when the browser tab is closed, causing “Bad state: No element” errors and breaking the DWDS connection.
+- [flutter/173770](https://github.com/flutter/flutter/issues/173770) Mitigates a memory leak that occurs on Android when Activities are not kept and an Activity is exited and re-entered.
+
+### [3.38.0-0.1.pre](https://github.com/flutter/flutter/releases/tag/3.38.0-0.1.pre)
+
+#### Framework
+* Allow OverlayPortal.overlayChildLayoutBuilder to choose root Overlay by @chunhtai in https://github.com/flutter/flutter/pull/174239
+* [ Widget Preview] Add `group` property to `Preview` by @bkonyi in https://github.com/flutter/flutter/pull/174849
```

Comment:
Nit: More weird spacing. I realize these weren't all written by you but I guess it makes sense to clean them up here.

It seems that below some comments put spaces around square brackets and some don't, so it might make sense to standardize.